data/human/deep missions.txt

```diff
@@ -1214,7 +1214,7 @@ mission "Deep: Project Hawking: Prime"
 		ship "Marauder Raven (Engines)" "Krampus"
 	on offer
 		conversation
-			`As you approach the massive <planet> spaceport building, Pierre directs you to park on a landing pad on the third story close to the warehouse section. "We're picking up various materials used in the creation of ships that we need samples of for testing," Pierre explains.`
+			`As you approach the massive <planet> spaceport building, Pierre directs you to park on a landing pad on the third story, close to the warehouse section. "We're picking up various materials used in the creation of ships, that we need samples of for testing," Pierre explains.`
```

Comment:
I'm expecting this line to be challenged, so this is why I've made the change:
You can either say, "We're picking up various materials that we need samples of for testing," or "We're picking up various materials used in the creation of ships."
I think the original makes it sound like they need samples of ships, which isn't wrong, but they're not taking the materials from ships, they're taking the materials before they become part of a ship.